docs/install.md

```diff
@@ -5,78 +5,99 @@ feature_maturity: stable
 enterprise: 'no'
 ---
 
-Spark is available in the Universe and can be installed by using either the web interface or the DC/OS CLI.
+Spark is available in the Universe and can be installed by using either the GUI or the DC/OS CLI.
 
-##  <a name="install-enterprise"></a>Prerequisites
```

Comment:
it was not referenced anywhere: https://github.com/mesosphere/spark-build/search?utf8=%E2%9C%93&q=%22install-enterprise%22&type= or https://github.com/mesosphere/universe/search?utf8=%E2%9C%93&q=%22install-enterprise%22&type=